Krakenforce_e_project_sem1/Source Code/connection/connect.php

```diff
@@ -0,0 +1,53 @@
+<?php
+class ShowPhp{
+    static function showMessage($message){
+        echo "<script>alert(\"$message\");</script>";
+    }
+}
+class Database{
+    private $dns = "mysql:host=localhost; dbname=e_sem1; charset=utf8";
+    private $username = "krakenforce";
+    private $password = "123456";
+    private $pdo;
+    private $stmt;
+    public function __construct(){
+        try{
+            $this->pdo = new PDO($this->dns, $this->username, $this->password);
```

Comment:
thiếu dòng: 
$this->pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION)
để bật catch exception  cũa pdo. đã thêm tại branch tuan.